entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -l
+
+php --version
+aws --version
+
+# Set the ssh key name.
+SSH_KEY_NAME="acquia"
+
+# Configure the known hosts.
+ssh-keyscan -t rsa "$INPUT_ACQUIA_PROJECT.ssh.prod.acquia-sites.com" >> ~/.ssh/known_hosts
```

Comment:
@markdorison this and any other lines relating to SSH could use your eyes as I wasn't 100% sure if this was best practice. I was just trying to get it to work.

Reply:
I am not exactly sure of this BUT...

I believe that doing it this way leaves the action vulnerable to a MITM attack. I think you want to have you determine the known host ahead of time and then pass it to the action. That way, if the host changes between when they set it up and when it is run (or the 100th time it is run) it will fail.